src/Compilers/VisualBasic/Portable/xlf/VBResources.ja.xlf

```diff
@@ -8395,7 +8396,7 @@
 -vbruntime:&lt;file&gt;                 Compile with the alternate Visual Basic
                                   runtime in &lt;file&gt;.
 </source>
-        <target state="translated">                  Visual Basic コンパイラのオプション
+        <target state="needs-review-translation">                  Visual Basic コンパイラのオプション
```

Comment:
Do I need to go through and revert these translations that don't appear to have changed?